README.rst

```diff
@@ -20,7 +20,7 @@ Features
 ========
 
 Use simple template tags to generate icons in your web application.
-Supports Font Awesome out of the box, easily adaptable for other icon libraries.
+Supports Font Awesome and Icons8 out of the box, easily adaptable for other icon libraries.
```

Comment:
It would be nice to let this sentence reflect the current state of the project, bootstrap3 and material are supported too.

Reply:
I pushed an update to reflect your suggestion